### PR TITLE
Unify description for different elements gridexplore

### PR DESCRIPTION
**Pull request by souissimai**

### CODE REVIEW

File: src/translations/fr.json [hunk 1]
```diff
@@ -161,6 +161,7 @@
     "FreeProps": "Propriétés de l'ouvrage",
     "SubstationFreeProps": "Propriétés du site de l'ouvrage",
     "AddFreePropCrit": "Ajouter un filtrage par propriété",
+    "AddDescription": "Ajouter une description (optionnel)",
```
Comment (klesaulnier) [inline src/translations/fr.json:165]: No need to create a new label when "descriptionProperty" was used for the same thing

**Review by klesaulnier — APPROVED** [review]
Test: OK
Code: remove duplicate

**Review by AAJELLAL — APPROVED** [review]

File: src/translations/fr.json (hunk 1 above)
Comment (souissimai) [reply src/translations/fr.json:165]: Description Property:  is the label used to describe the purpose or content of an item.

AddDescription  This label will be attached to a clickable link that users can interact with to add a description.

**Comment by souissimai** [comment]
Test: OK
Code: some minor remarks

**Pull request merged**